### ISSUE TITLE
Setting the base price of new buildings

### ISSUE THREAD

**Issue by DPS2004** [klattmose/klattmose.github.io#24]
Hello,

I am currently working on a mod that adds a new building and am basing it off of the code of the Black Hole Inverter mod to get started. I am a bit confused about how the base price of the building is set when making it? Where I would expect to find the price, I found `"doesn't matter what you put here",`.

**Comment by klattmose** [comment]
The game will automatically calculate the base price and cps of a new building. I put ```"doesn't matter what you put here"``` because it really doesn't. Any argument in that spot gets ignored. 

To find the game's calculation, go to main.js and search for "new automated price and CpS curves" (without quotes)

**Comment by DPS2004** [comment]
I think I get it. If by default modded buildings are added after idleverse, is there a way to make the building appear earlier in the game? like, if I want my building to take Bank's place, and move bank and all the other buildings after it up in the price curve

**Comment by klattmose** [comment]
You would need to make sure each object has the right id, recalculate the prices, rebuild Game.ObjectsById, rerun Game.BuildStore, and there would *still* be problems with the store showing out of order, the rows in the center might get wonky, and the tiered upgrades would be the wrong price, and that's not even getting into issues with saving and loading. There may be more problems that I haven't thought of.

Basically, it's going to be a lot more effort than it seems.
